src/v/cloud_storage/partition_manifest.cc

```diff
@@ -731,7 +732,7 @@ void partition_manifest::update(const partition_manifest_handler& handler) {
         throw std::runtime_error(fmt_with_ctx(
           fmt::format,
           "partition manifest version {} is not supported",
-          handler._version));
+          handler._version.value()));
```

Comment:
this looks out of place in this commit. what's happening here?

Reply:
Reverted that but included `utils/to_string.h`. I think that I added that because it stopped compiling after I removed some header (to_string was a transient dependency).